src/overlays/actors/ovl_En_Po_Composer/z_en_po_composer.h

```diff
@@ -2,16 +2,58 @@
 #define Z_EN_PO_COMPOSER_H
 
 #include "global.h"
+#include "objects/object_po_composer/object_po_composer.h"
+
+#define PO_COMPOSER_IS_FLAT(actor) ((actor)->params & 0x8000)
+
+typedef enum {
+    /*  1 */ COMPOSER_CS_ACTION_IDLE = 1,
+    /*  2 */ COMPOSER_CS_ACTION_APPEAR_SPIN,
```

Comment:
```suggestion
    /*  1 */ COMPOSER_CUEID_IDLE = 1,
    /*  2 */ COMPOSER_CUEID_APPEAR_SPIN,
```
CsAction was renamed to CueId in a cutscene docs pass